### ISSUE TITLE
 pretrained models for evaluation

### ISSUE THREAD

**Issue by yzfly** [Phoenix1327/tea-action-recognition#2]
Do you have pretrained models available for evaluation?

**Comment by White1973** [comment]
I can't find the pretrained models, could you tell me where I can get the models.

**Comment by yzfly** [comment]
The author didn't share them.  Maybe you should prepare the datasets and train it yourself. I have trained the 8 segments model and got `Best Prec@1: 48.82` on somthingsomething-v1.

**Comment by meifish** [comment]
In `tea50_8f.py` there are several models listed, but it seems only `res2net50_26w_4s` can be curled, and all the others seem to be dummy, or it is just me not able to access it? 

Thanks!

**Comment by Phoenix1327** [comment]
> In `tea50_8f.py` there are several models listed, but it seems only `res2net50_26w_4s` can be curled, and all the others seem to be dummy, or it is just me not able to access it?
> 
> Thanks!

The authors of Res2Net have updated the urls of pertained models. The new urls can be found in [https://github.com/Res2Net/Res2Net-PretrainedModels/blob/master/res2net.py](url).

**Comment by meifish** [comment]
> The author didn't share them. Maybe you should prepare the datasets and train it yourself. I have trained the 8 segments model and got `Best Prec@1: 48.82` on somthingsomething-v1.

Thank you for sharing your trained accuracy. I wonder how many GPUs and the batch size you adopted in your training? 
I got a relatively low accuracy with batch-size = 8 ( trained on something-something-v2 sampled dataset ).

**Comment by yzfly** [comment]
> > The author didn't share them. Maybe you should prepare the datasets and train it yourself. I have trained the 8 segments model and got `Best Prec@1: 48.82` on somthingsomething-v1.
> 
> Thank you for sharing your trained accuracy. I wonder how many GPUs and the batch size you adopted in your training?
> I got a relatively low accuracy with batch-size = 8 ( trained on something-something-v2 sampled dataset ).

I did not train on something-something-v2 dataset. You can reference to [issues4](https://github.com/Phoenix1327/tea-action-recognition/issues/4) for training settings.

**Comment by WAKEYOU** [comment]
> > In `tea50_8f.py` there are several models listed, but it seems only `res2net50_26w_4s` can be curled, and all the others seem to be dummy, or it is just me not able to access it?
> > Thanks!
> 
> The authors of Res2Net have updated the urls of pertained models. The new urls can be found in [https://github.com/Res2Net/Res2Net-PretrainedModels/blob/master/res2net.py](url).

Thanks. But when I open the link, it says it doesn't match your search results. Could you please update the link again？

**Comment by Phoenix1327** [comment]
> > > In `tea50_8f.py` there are several models listed, but it seems only `res2net50_26w_4s` can be curled, and all the others seem to be dummy, or it is just me not able to access it?
> > > Thanks!
> > 
> > 
> > The authors of Res2Net have updated the urls of pertained models. The new urls can be found in [https://github.com/Res2Net/Res2Net-PretrainedModels/blob/master/res2net.py](url).
> 
> Thanks. But when I open the link, it says it doesn't match your search results. Could you please update the link again？

[Res2Net](https://github.com/Res2Net/Res2Net-PretrainedModels/blob/master/res2net.py)

**Comment by sudonto** [comment]
Hello @yzfly @Phoenix1327 , do you have TEA model pretrained on Kinetics 400 dataset?

**Comment by zzx904** [comment]
Hello @yzfly @Phoenix1327 , do you have TEA model pretrained on Kinetics 400 dataset? I want to fine-tune it on UCF.
